xl2times/transforms.py

```diff
@@ -552,7 +584,8 @@ def process_user_constraint_table(
         # TODO: apply table.uc_sets
 
         # Fill in UC_N blank cells with value from above
-        df["uc_n"] = df["uc_n"].ffill()
+        if "uc_n" in df.columns:
+            df["uc_n"] = df["uc_n"].ffill()
```

Comment:
`uc_n` should always be there (in this table type). I've expanded the meta data in veda-tags.json recently that can be used to discard invalid rows (due to missing info) systematically accross all table types.

Reply:
Ah cool, then I've probably merged in your changes since I fixed this here.  Will revert this too.